src/components/Clock.tsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+type Props = {
+  clockName: string;
+  hasClock: boolean;
+};
+
+type State = {
+  date: Date;
+};
+
+export default class Clock extends React.Component<Props, State> {
+  state = {
+    date: new Date(),
+  };
+
+  componentDidMount() {
+    window.setInterval(
+      () => this.tick(),
+      1000,
+    );
+  }
+
+  tick() {
+    this.setState({
+      date: new Date(),
+    });
+    if (this.props.hasClock === true) {
+      console.info(this.state.date.toUTCString().slice(-12, -4));
+    }
+  }
+
+  render() {
+    const { date } = this.state;
+    const { hasClock, clockName } = this.props;
+
+    return (
+      <div className={`${hasClock === true ? 'Clock' : 'ClockHide'}`}>
```

Comment:
let's use [classnames](https://www.npmjs.com/package/classnames)

Reply:
I already used classnames. Did you mean not to use them?